packages/suite/src/views/settings/device/index.tsx

```diff
@@ -72,11 +71,7 @@ const Settings = ({
     }, [device]);
 
     if (!device?.features) {
-        return (
-            <SuiteLayout title="Settings" secondaryMenu={<SettingsMenu />}>
-                no device connected
-            </SuiteLayout>
-        );
+        return <SettingsLayout>no device connected</SettingsLayout>;
```

Comment:
missing translation

Reply:
no design provided for this. we still dont know how settings should appear if device is disconnected. generally I tend not to provide translations if designs are not ready as it might be useless for me and translators. @matejzak

Reply:
but I am going to add it there ofc :D